packages/ui/src/partials/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './card'
 export * from './checkbox'
 export * from './input'
 export * from './label'
+export * from './react-form/react-form'
```

Comment:
```suggestion
export * from './react-form/form'
```

will it be better to rename the form comp as form?

Reply:
This can be done